tests/test_utils/tests.py

```diff
@@ -627,6 +627,8 @@ def test_parsing_errors(self):
             self.assertHTMLEqual('<p>', '')
         with self.assertRaises(AssertionError):
             self.assertHTMLEqual('', '<p>')
+        with self.assertRaises(AssertionError):
```

Comment:
Could you use `assertRaisesMessage` to check the message that the code generates as well?